lib/src/ui/screen/review_screen.dart

```diff
@@ -10,12 +16,64 @@ class ReviewScreen extends StatefulWidget {
 class ReviewScreenState extends State<ReviewScreen> {
   @override
   Widget build(BuildContext context) {
+    reviewBloc.fetchCurrentReview();
     return Scaffold(
-      appBar: AppBar(
-        title: Text("Review Screen"),
-      ),
-      drawer: ListViewDrawerBloc().getDefaultListDrawer(),
-      body: Text("Reviews"),
-    );
+        appBar: AppBar(
+          title: Text("Review Screen"),
+        ),
+        drawer: ListViewDrawerBloc().getDefaultListDrawer(),
+        body: StreamBuilder(
+          stream: reviewBloc.currentReview,
+          builder: (context, AsyncSnapshot<ReviewModel> snapshot) {
+            if (snapshot.hasData) {
+              return fullReview(snapshot);
+            } else if (snapshot.hasError) {
+              return Text(snapshot.error.toString());
+            }
+            return Center(child: CircularProgressIndicator());
+          },
+        ));
   }
 }
+
+Widget fullReview(AsyncSnapshot<ReviewModel> snapshot) {
+  return Container(
+      child: Column(crossAxisAlignment: CrossAxisAlignment.center, children: [
+        SizedBox(height: 10),
+        Text(
+          snapshot.data.location.locationName,
+          style: TextStyle(fontSize: 30.0),
```

Comment:
styles should use the main theme styles (or be added to them if not already there) in theme/main_theme.dart